backend-implicaction/src/test/java/com/dynonuggets/refonteimplicaction/controller/ExperienceControllerTest.java

```diff
@@ -0,0 +1,226 @@
+import org.springframework.test.web.servlet.ResultActions;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import static com.dynonuggets.refonteimplicaction.utils.ApiUrls.DELETE_EXPERIENCES_URI;
+import static com.dynonuggets.refonteimplicaction.utils.ApiUrls.EXPERIENCES_BASE_URI;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.*;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@WebMvcTest(controllers = ExperienceController.class)
+class ExperienceControllerTest extends ControllerIntegrationTestBase {
+
+    @MockBean
+    WorkExperienceService experienceService;
+    ArrayList<String> roles = new ArrayList<>();
+    UserDto user;
+
+    @BeforeEach
+    void setUp() {
+        roles.add(RoleEnum.USER.getLongName());
+        user = UserDto.builder()
+                .id(3L)
+                .username("paul-sdv")
+                .firstname("Paul")
+                .lastname("Flu")
+                .email("paul@implicaction.fr")
+                .url("www.google.fr")
+                .hobbies("surf,gaming,judo")
+                .purpose("")
+                .registeredAt(null)
+                .activatedAt(null)
+                .roles(roles)
+                .active(true)
+                .build();
+    }
+
+    @Test
+    @WithMockUser
+    void should_create_experience_when_authenticated() throws Exception {
+        // given
+        WorkExperienceDto experienceDto = WorkExperienceDto.builder()
+                .user(user)
+                .companyName("Idemia")
+                .description("Creation")
+                .startedAt(null)
+                .finishedAt(null)
+                .label("Label")
+                .build();
+
+        String json = gson.toJson(experienceDto);
+
+        WorkExperienceDto expectedDto = WorkExperienceDto.builder()
+                .id(123L)
+                .companyName("Idemia")
+                .description("Creation")
+                .startedAt(null)
+                .finishedAt(null)
+                .label("Label")
+                .build();
+
+
+        given(experienceService.saveOrUpdateExperience(any())).willReturn(expectedDto);
+
+        // when
+        final ResultActions resultActions = mvc.perform(
+                post(EXPERIENCES_BASE_URI).content(json).accept(APPLICATION_JSON).contentType(APPLICATION_JSON)
+        );
+
+        // then
+        resultActions.andDo(print())
+                .andExpect(status().isCreated())
+                .andExpect(content().contentType("application/json"))
```

Comment:
```suggestion
                .andExpect(content().contentType(APPLICATION_JSON_VALUE))
```
C'est le top quand on utilise des constantes 😉